apps/mobile-wallet/src/components/layout/BottomModal.tsx

```diff
@@ -213,10 +224,14 @@ const BottomModal = ({ Content, isOpen, onClose, maximisedContent, customMinHeig
             <HandleContainer>
               <Handle style={handleAnimatedStyle} />
             </HandleContainer>
-            <ContentContainer>
-              <Navigation style={modalNavigationAnimatedStyle}>
-                <Button onPress={handleClose} iconProps={{ name: 'close-outline' }} round />
-              </Navigation>
+            <Navigation style={modalNavigationAnimatedStyle}>
+              <NavigationButtonContainer align="left" />
+              {title && <Title semiBold>{title}</Title>}
+              <NavigationButtonContainer align="right">
+                <Button onPress={handleClose} iconProps={{ name: 'x' }} round compact />
```

Comment:
Since we are at it, wouldn't it better to reuse the `CloseButton` which we are already using in all modals that implement `ModalWithBackdrop` as well as `AddressScreen` and `DestinationScreen` for a more consistent UI?

We also have to keep in mind Google's accessibility recommendation when it comes to the clickable area of buttons which (IIRC) needs to be at least 48dp. That doesn't mean that the visual button needs to be that big, but the clickable area (example: padding around it). We have a lot of buttons that need to be improved.

At least, if we reuse the same buttons, we can fix them all at once.

<img width="1688" alt="image" src="https://github.com/user-attachments/assets/825d1421-9b4d-4ab6-8450-8f8b0a7a6720">

Reply:
That's a good point. Thanks.

Reply:
Done